### PR TITLE
feat: migrate to gotocompany

### PR DESCRIPTION
**Pull request by rahmatrhd**

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -22,12 +22,12 @@ jobs:
         run: cd docs && yarn build
       - name: Deploy docs
         env:
-          GIT_USER: ravisuhag
+          GIT_USER: mabdh
```
Comment (mabdh) [inline .github/workflows/docs.yaml:25]:
```suggestion
          GIT_USER: anshuman-gojek
```

File: .github/workflows/docs.yaml [hunk 2]
```diff
@@ -22,12 +22,12 @@ jobs:
...
           GIT_PASS: ${{ secrets.DOCU_RS_TOKEN }}
           DEPLOYMENT_BRANCH: gh-pages
           CURRENT_BRANCH: main
         working-directory: docs
         run: |
-          git config --global user.email "suhag.ravi@gmail.com"
-          git config --global user.name "ravisuhag"
+          git config --global user.email "muhammad.abduh@gojek.com"
+          git config --global user.name "Muhammad Abduh"
           yarn deploy
```
Comment (mabdh) [inline .github/workflows/docs.yaml:33]:
```suggestion
          git config --global user.email "anshuman.srivastava@gojek.com"
          git config --global user.name "anshuman-gojek"
          yarn deploy
```

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -9,6 +9,11 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+      - uses: extractions/netrc@v1 # to work with private repo dependencies
+        with:
+          machine: github.com
+          username: ${{ secrets.USERNAME }}
+          password: ${{ secrets.PASSWORD }}
```
Comment (mabdh) [inline .github/workflows/release.yaml:16]: we can remove this since repo is public
```suggestion
```

File: .github/workflows/release.yaml [hunk 4]
```diff
@@ -23,6 +28,11 @@ jobs:
           registry: docker.io
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - uses: extractions/netrc@v1 # to work with private repo dependencies
+        with:
+          machine: github.com
+          username: ${{ secrets.USERNAME }}
+          password: ${{ secrets.PASSWORD }}
```
Comment (mabdh) [inline .github/workflows/release.yaml:35]: same with this
```suggestion
```

File: .github/workflows/release.yaml [hunk 5]
```diff
@@ -31,3 +41,4 @@ jobs:
           args: --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GO_RELEASER_TOKEN }}
+          GOPRIVATE: ${{ env.GOPRIVATE }}
```
Comment (mabdh) [inline .github/workflows/release.yaml:44]:
```suggestion
```

File: .github/workflows/test.yaml [hunk 6]
```diff
@@ -10,6 +10,11 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: "1.18"
+      - uses: extractions/netrc@v1 # to work with private repo dependencies
+        with:
+          machine: github.com
+          username: ${{ secrets.USERNAME }}
+          password: ${{ secrets.PASSWORD }}
```
Comment (mabdh) [inline .github/workflows/test.yaml:17]:
```suggestion
```

File: .github/workflows/test.yaml [hunk 7]
```diff
@@ -28,13 +33,14 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ matrix.go-version }}
+      - uses: extractions/netrc@v1 # to work with private repo dependencies
+        with:
+          machine: github.com
+          username: ${{ secrets.USERNAME }}
+          password: ${{ secrets.PASSWORD }}
```
Comment (mabdh) [inline .github/workflows/test.yaml:40]:
```suggestion
```

File: .github/workflows/test.yaml [hunk 8]
```diff
@@ -28,13 +33,14 @@ jobs:
...
         uses: actions/checkout@v2
       - name: Test
         run: make test
-      - name: Install goveralls and send coverage
         env:
-          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          go install github.com/mattn/goveralls@latest
-          goveralls -coverprofile=coverage.out -service=github
+          GOPRIVATE: ${{ env.GOPRIVATE }}
```
Comment (mabdh) [inline .github/workflows/test.yaml:46]:
```suggestion
```

File: .github/workflows/test.yaml [hunk 9]
```diff
@@ -28,13 +33,14 @@ jobs:
...
+        with:
+          machine: github.com
+          username: ${{ secrets.USERNAME }}
+          password: ${{ secrets.PASSWORD }}
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Test
         run: make test
-      - name: Install goveralls and send coverage
         env:
```
Comment (mabdh) [inline .github/workflows/test.yaml:45]: It is okay to have this now

```suggestion
      - name: Install goveralls and send coverage
        env:
          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        run: |
          go install github.com/mattn/goveralls@latest
          goveralls -coverprofile=coverage.out -service=github
```

File: .goreleaser.yaml [hunk 10]
```diff
@@ -81,5 +81,5 @@ brews:
     install: |-
       bin.install "guardian"
     commit_author:
-      name: Ravi Suhag
-      email: suhag.ravi@gmail.com
+      name: Muhammad Abduh
+      email: muhammad.abduh@gojek.com
```
Comment (mabdh) [inline .goreleaser.yaml:85]:
```suggestion
      name: github-actions[bot]
      email: 41898282+github-actions[bot]@users.noreply.github.com
```

File: NOTICE [hunk 11]
```diff
@@ -0,0 +1,5 @@
+Change(s)
```
Comment (mabdh) [inline NOTICE:1]: @rahmatrhd we can remove this

Comment (bsushmith) [reply NOTICE]: Done

**Review by mabdh — APPROVED** [review]

**Review by bsushmith — APPROVED** [review]

**Pull request merged**